### DISCUSSION TITLE
📢 New Release Alert: op-stack v1.7.4!

### DISCUSSION THREAD

**Discussion by ZakAyesh** [ethereum-optimism/developers#371] [Announcements]
📢 New Release Alert: op-stack v1.7.4!
We are thrilled to announce the release of op-stack v1.7.4, which includes crucial updates and enhancements across op-node, op-batcher, and op-proposer.

What’s New?
🐞 op-node Bug Fix: A significant fix has been implemented for op-node. Previously, if an L1 block got reorganized out during blob retrieval, op-node could enter a loop, endlessly retrieving a blob that will never exist, requiring a restart. This issue has been resolved by internally signaling the correct error types, which forces a derivation pipeline reset under such circumstances.

✨ op-batcher & op-proposer Synchronization Enhancements: These components can now delay their operations until the sequencer has synced to the current L1 head. This update addresses a potential issue where, after a simultaneous restart of op-batcher/proposer and op-node, duplicate batches could be resent from the last finalized L2 block because a freshly restarted op-node resyncs from the finalized head, potentially signaling a prematurely safe head in its sync status. This feature is off by default so we recommend testing it using the new batcher and proposer flag ​--wait-node-sync​ detailed below.

 🧑‍💻New Configuration Options:
--wait-node-sync: Allows op-batcher and op-proposer to wait for the sequencer’s verifier confirmation depth of typically 4 L1 blocks, or approximately 1 minute, at startup.

--check-recent-txs-depth: An optional flag that enables the batcher to check for recent batcher transactions to determine a potentially earlier sync target.  This is an optimization to counter the 1-minute startup delay from the ​wait-node-sync​ flag. This feature is off by default and should be set to the sequencer’s verifier confirmation depth to get enabled.

For a detailed view of all changes and to contribute further, please visit our GitHub.
https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.4

We have also created a new Telegram channel you can join where we post all of the latest node and protocol updates. See you there!
https://t.me/+vPoESXHxEm83YWQx

**Comment by Chomtana** [comment]
[v1.7.5](https://github.com/ethereum-optimism/optimism/releases/tag/op-node%2Fv1.7.5) has been released?

**Reply by ZakAyesh** [reply]
@Chomtana Yes just posted the announcement for it in this channel :).
